pact/Pact/Core/IR/Eval/Direct/Evaluator.hs

```diff
@@ -153,6 +154,26 @@ eval purity benv term = do
     VPactValue pv -> pure pv
     _ -> throwExecutionError (view termInfo term) (EvalError "Evaluation did not reduce to a value")
 
+
+evalWithinCap
+  :: forall e b i
+  .  (IsBuiltin b)
+  => i
+  -> Purity
+  -> BuiltinEnv e b i
+  -> CapToken QualifiedName PactValue
+  -> EvalTerm b i
+  -> EvalM e b i PactValue
+evalWithinCap info purity benv (CapToken qualName vs) term = do
+  ee <- viewEvalEnv id
+  (_, mh) <- getDefCapQN info qualName
+  let ct = CapToken (qualNameToFqn qualName mh) vs
```

Comment:
Maybe we should go with only FQN?

Reply:
See: Other comment about this same change. This is for external facing apis